src/v2/providers/identity.ts

```diff
@@ -50,6 +57,12 @@ export function beforeUserCreated(
     | void
     | Promise<void>
 ): BlockingFunction;
+
+/**
+ * Handles an event that is triggered before a user is created
+ * @param optsOrHandler - Either an object containing Function options, or an event handler (run before user creation)
+ * @param handler? - If defined, an wvent handler which is run every time before a user is created
```

Comment:
Typo, "event"

Also, should "Function" be capped?  Please check throughout, thanks!

Reply:
Updated, I think it should be lowercased, updated throughout